packages/web-components/fast-components/src/radio-group/radio-group.template.ts

```diff
@@ -8,7 +8,13 @@ export const RadioGroupTemplate = html<RadioGroup>`
         aria-readonly="${x => x.readOnly}"
     >
         <slot name="label"></slot>
-        <div class="positioning-region" part="positioning-region">
+        <div
+            class="positioning-region ${x =>
+                x.orientation === RadioOrientation.horizontal
+                    ? "horizontal"
+                    : "vertical"}"
+            part="positioning-region"
+        >
```

Comment:
I think we'll want this class on the host element.

Reply:
I can do that but it would potentially mess up the label slot

Reply:
I think if we have the label slot we don't necessarily want it to fall in the flex of the radios so the orientation is in a good place where it's at now. Unless you always want the label to be inline but in the horizontal (default) case that doesn't seem like what we'd want?

Reply:
going to resolve this, I don't see any objection